jobwatch/hourly_watch.py

```diff
@@ -164,7 +164,9 @@ def main():
 
     if args.jobs == 'ska':
         jws = [
-            SkaURLWatch('kadi', 1, 'http://kadi.cfa.harvard.edu')
+            SkaURLWatch('kadi', 1, 'https://kadi.cfa.harvard.edu'),
+            SkaFileWatch('kadi', 1, 'cmds.h5'),
```

Comment:
This is kadi commands V1 which I'm hoping will be dropped soon. The V2 commands are `cmds2.h5`, however this only updates when new commands are added, which means typically the file will be up to a week old.

Reply:
Oops, yeah I confused them when looking at recently updated files in the directory.  I'll rethink this.